### PR TITLE
Deployment History page and Server status checking

### PR DESCRIPTION
**Pull request by angelamancini**
Added Deployment History page to see past deployments.
Updated tagging gem
Old servers now terminate after all new instances are operational

### CODE REVIEW

File: config/github_config.yml.tmpl [hunk 1]
```diff
@@ -1,4 +1,4 @@
 development:
```
Comment (erumble) [inline config/github_config.yml.tmpl]: shoudl these be changed to `@@RAILS_ENV@@:`?

File: config/rightscale_config.yml.tmpl [hunk 2]
```diff
@@ -1,16 +1,5 @@
 development:
```
Comment (erumble) [inline config/rightscale_config.yml.tmpl]: Should this be `@@RAILS_ENV@@:`?

File: lib/modules/rightscale/rightscale.rb [hunk 3]
```diff
@@ -92,4 +101,23 @@ def terminate(array_id, environment, deployment_start)
       end
     end
   end
+  class Server
```
Comment (erumble) [inline lib/modules/rightscale/rightscale.rb]: I'm not a big fan of defining more than 1 class in a single file. I would prefer to see a folder with the module name, and then each class that belongs to that module as an individual file in that folder. Keeping the module namespace. Just a bit easier to see where things are later on.

**Comment by erumble** [comment]
Aside from the comment about classes being in their own file, it looks good. :shipit:

File: app/controllers/deployments_controller.rb [hunk 4]
```diff
@@ -4,6 +4,11 @@ class DeploymentsController < ApplicationController
   include Github
   include ActionView::Helpers::TextHelper
 
+  def index
+    @deployment_profile = DeploymentProfile.find(params[:deployment_profile_id])
+    @deployments = Deployment.where(deployment_profile_id: params[:deployment_profile_id]).order(created_at: :desc)
```
Comment (Sjeanpierre) [inline app/controllers/deployments_controller.rb]: is it possible to do @deployment_profile.deployments?

**Pull request merged**